glide.yaml

```diff
@@ -1,6 +1,6 @@
 package: github.com/uber/ringpop-go
 import:
-- package: github.com/Sirupsen/logrus
+- package: github.com/sirupsen/logrus
```

Comment:
@yulunli github.com/uber-common/bark was fixed on June 16th by @akshayjshah so this should be okay now. I think you'll just need to update the diff to make sure the glide.lock file is still okay.